src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/sections/genes/Genes.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useCallback } from 'react';
+import { connect } from 'react-redux';
+import { RootState } from 'src/store';
+
+import { getGeneAttributes } from '../../state/attributesAccordionSelector';
+import { setGeneAttributes } from '../../state/attributesAccordionActions';
+import CheckBoxGrid, {
+  filterCheckedAttributes
+} from 'src/content/app/custom-download/components/checkbox-grid/CheckboxGrid';
+
+import styles from './Genes.scss';
+
+type ownProps = {
+  hideUnchecked?: boolean;
+  hideTitles?: boolean;
+};
+
+type Props = ownProps & StateProps & DispatchProps;
+
+const Genes = (props: Props) => {
+  const onChangeHandler = useCallback(
+    (status: boolean, subSection: string, attributeId: string) => {
+      if (!props.geneAttributes) {
+        return;
+      }
+      const newGeneAttributes = { ...props.geneAttributes };
+
+      newGeneAttributes[subSection][attributeId].checkedStatus = status;
+
+      props.setGeneAttributes(newGeneAttributes);
+    },
+    [props.geneAttributes]
+  );
+
+  if (props.hideUnchecked) {
+    if (!props.geneAttributes) {
+      return null;
+    }
+    const checkedAttributes = filterCheckedAttributes(props.geneAttributes);
+
+    if (Object.keys(checkedAttributes).length === 0) {
+      return null;
+    }
+
+    return (
+      <div className={styles.checkboxGridWrapper}>
+        <CheckBoxGrid
```

Comment:
Checkbox is one word (and in other files you do not capitalize the Box)

Reply:
That's now renamed